src/hotspot/share/jfr/recorder/checkpoint/jfrCheckpointWriter.cpp

```diff
@@ -78,7 +78,7 @@ static void write_checkpoint_header(u1* pos, int64_t size, jlong time, u4 checkp
   JfrBigEndianWriter be_writer(pos, sizeof(JfrCheckpointEntry));
   be_writer.write(size);
   be_writer.write(time);
-  be_writer.write(JfrTicks::now().value() - time);
```

Comment:
Why is this changed?

Reply:
It was a small optimization attempted before the current system of writing virtual thread checkpoints in bulk. Will restore it, thank you.